inbm/dispatcher-agent/dispatcher/schedule/rpc_activate_operation.py

```diff
@@ -0,0 +1,31 @@
+"""
+    Executes the AMT Rpc Activation command.
+
+    Copyright (C) 2024 Intel Corporation
+    SPDX-License-Identifier: Apache-2.0
+"""
+from typing import Optional
+from inbm_common_lib.shell_runner import PseudoShellRunner
+
+
+
+class RpcActivateOperation: 
+    def __init__(self, url: str, profile_name: str) -> str:
```

Comment:
python constructor should always return `None`

Reply:
Done.